clang/lib/DPCT/APINames.inc

```diff
@@ -1407,8 +1407,8 @@ ENTRY(cuTexRefSetMipmapLevelClamp, cuTexRefSetMipmapLevelClamp, false, NO_FLAG,
 ENTRY(cuTexRefSetMipmappedArray, cuTexRefSetMipmappedArray, false, NO_FLAG, P4, "comment")
 ENTRY(cuTexRefCreate, cuTexRefCreate, false, NO_FLAG, P4, "comment")
 ENTRY(cuTexRefDestroy, cuTexRefDestroy, false, NO_FLAG, P4, "comment")
-ENTRY(cuMemAdvise, cuMemAdvise, true, NO_FLAG, P4, "Successful for restricted USM")
-ENTRY(cuMemPrefetchAsync, cuMemPrefetchAsync, true, NO_FLAG, P4, "Successful: USM only")
+ENTRY(cuMemAdvise, cuMemAdvise, true, NO_FLAG, P4, "DPCT1063/DPCT1007")
```

Comment:
let's remove 1007 here, which is conflict with 3rd field.  It's known that migration maybe differenct when using differnt options of dpct, and some option may result the migration of  API is not support yet.